docs/articles/documentation/reference/test-api/testcontroller/switchtopreviouswindow.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: docs
+title: t.switchToPreviousWindow Method
+permalink: /documentation/reference/test-api/testcontroller/switchtopreviouswindow.html
+---
+
+# t.switchToPreviousWindow Method
+
+Activates the most recent of the previously active windows. If no new windows are open or closed, consecutive calls of the method cycle back and forth between the two most recent windows.
```

Comment:
> If no new windows are open or closed, consecutive calls of the method cycle back and forth between the two most recent windows.

This part could go just before the example, after the syntax part. Cause the "short description" is just an idea, no details required here.